packages/gradle/src/plugin/dependencies.ts

```diff
@@ -89,7 +90,7 @@ function processGradleDependencies(
   context: CreateDependenciesContext
 ): Set<RawProjectGraphDependency> {
   const dependencies: Set<RawProjectGraphDependency> = new Set();
-  const lines = readFileSync(depsFile).toString().split('\n');
+  const lines = readFileSync(depsFile).toString().split(newLineSeparator);
```

Comment:
are there any guarantees that this file will have `\r\n` on windows?
I'm asking cause depending on the nature of this file - it also might use `\n` even on Windows (for example if it lives under the git source control with configured `autocrlf=input`).
As a more robust alternative - `/\r?\n/` regex might be used